src/data/markdown/translated-guides/en/02 Using k6/02 Metrics.md

```diff
@@ -69,16 +69,80 @@ All the `http_req_...` lines and the ones after them are _built-in_ metrics that
 
 The following _built-in_ metrics will **always** be collected by k6:
 
-| Metric Name        | Type    | Description                                                                                                                                                                                                     |
-| ------------------ | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| vus                | Gauge   | Current number of active virtual users                                                                                                                                                                          |
-| vus_max            | Gauge   | Max possible number of virtual users (VU resources are pre-allocated, to ensure performance will not be affected when scaling up the load level)                                                                |
-| iterations         | Counter | The aggregate number of times the VUs in the test have executed the JS script (the `default` function).                                                                                                         |
-| iteration_duration | Trend   | The time it took to complete one full iteration of the default/main function.                                                                                                                                   |
+| Metric Name | Type | Description |
+| ----------- | ---- | ----------- |
+| vus                | Gauge   | Current number of active virtual users |
+| vus_max            | Gauge   | Max possible number of virtual users (VU resources are pre-allocated, to ensure performance will not be affected when scaling up the load level) |
+| iterations         | Counter | The aggregate number of times the VUs in the test have executed the JS script (the `default` function). |
+| iteration_duration | Trend   | The time it took to complete one full iteration. There is an available workaround for excluding the time spent in setup/teardown functions. Check the example below. |
```

Comment:
```suggestion
| iteration_duration | Trend   | The time it took to complete one full iteration. This includes the time spent in `setup` and `teardown` as well, there is currently a workaround shown in the example below. |
```
seems better to me to first say that setup and teardown time is included *and* then talk about a workaround